deploy/helm/sumologic/conf/traces/traces.otelagent.conf.yaml

```diff
@@ -1,8 +1 @@
-{{- $yamlFile := toYaml $.Values.otelagent.config }}
-
-{{- $otelcolService := include "sumologic.metadata.name.otelcol.service" . }}
-{{- $collectorAddress := printf "%s.%s" $otelcolService .Release.Namespace }}
-
-{{- $yamlFile := replace "exporters.otlp.endpoint.replace" $collectorAddress $yamlFile }}
-
-{{ tpl ($yamlFile | replace ": '{{" ": {{" | replace "}}'" "}}") . | nindent 2 }}
+{{ tpl (toYaml .Values.otelagent.config | replace ": '{{" ": {{" | replace "}}'" "}}") . | nindent 2 }}
```

Comment:
It is breaking change :disappointed:

Reply:
I think this file should be unchanged until release `3.0`

Reply:
Should I revert that file to previous version, the one with logic in conf file?

Reply:
yes:

![image](https://user-images.githubusercontent.com/58699848/158530777-a752653b-393b-40bb-9e36-f95730ce367c.png)

Reply:
Done. I've made it backward compatible.